cpp/src/arrow/json/parser_test.cc

```diff
@@ -188,7 +189,7 @@ TEST(BlockParserWithSchema, Nested) {
                       field("nuf", struct_({field("ps", utf8())}))},
                      {"[\"thing\", null, \"\xe5\xbf\x8d\", null]",
                       R"([["1", "2", "3"], ["2"], [], null])",
-                      R"([{"ps":null}, null, {"ps":"78"}, {"ps":"90"}])"});
+                      R"([{"ps":null}, {}, {"ps":"78"}, {"ps":"90"}])"});
```

Comment:
Hmm... @bkietz Can you validate this change is ok? It seems it only affects internal implementation details.

Reply:
This seems fine